web/packages/teleterm/src/ui/ConnectMyComputer/NavigationMenu.tsx

```diff
@@ -123,15 +129,16 @@ function getIndicatorStatus(
   const isAgentConfigured =
     isAgentConfiguredAttempt.status === 'success' &&
     isAgentConfiguredAttempt.data;
+  const isAgentCompatibilityUnkown = agentCompatibility === 'unknown';
 
-  if (!isAgentConfigured) {
+  if (!isAgentConfigured || isAgentCompatibilityUnkown) {
```

Comment:
What's the reason behind the check for agent compatibility here?

Reply:
Ah right, we don't need it.

BTW, I realized that this condition `!isAgentConfigured` causes the indicator to be not shown during the setup. 
I assume this is expected? I'm asking because I remember that I used your patch for this.